font-test-data/src/lib.rs

```diff
@@ -59,3 +59,26 @@ pub mod post {
         0x4, 0x68, 0x6f, 0x6c, 0x61, // 4, h o l a
     ];
 }
+
+pub mod cff2 {
+    /// CFF2 example font
```

Comment:
it's not a *font*, but a cff2 table right?

Reply:
Yes, it's a table. The spec referred to it as a font so I mindlessly copied it. Fixed.